sdks/java/harness/src/main/java/org/apache/beam/fn/harness/data/PCollectionConsumerRegistry.java

```diff
@@ -62,16 +70,18 @@
   @SuppressWarnings({"rawtypes"})
   abstract static class ConsumerAndMetadata {
     public static ConsumerAndMetadata forConsumer(
-        FnDataReceiver consumer, String pTransformId, SimpleExecutionState state) {
+        FnDataReceiver consumer, String pTransformId, SimpleExecutionState state, Coder coder) {
```

Comment:
```suggestion
        FnDataReceiver consumer, String pTransformId, SimpleExecutionState state, Coder valueCoder) {
```
`valueCoder` seems like less misleading to me. What do you think?